src/main/java/com/telefonica/iot/cygnus/handlers/OrionRestHandler.java

```diff
@@ -233,6 +233,12 @@ public void configure(Context context) {
             } // if else if
         } // while
         
+        // check if received content type is null
+        if (contentType == null) {
+            LOGGER.warn("Missing content type. Required xml or json.");
```

Comment:
I would say "Required application/json or application/xml", since it is the real content of a valid `Content-Type` header.